### ISSUE TITLE
Feature Request: Provide a way to set default values for new profiles

### ISSUE THREAD

**Issue by SwooshyCueb** [Kaldaien/SpecialK#3]

**Comment by Kaldaien** [comment]
Yeah, this is on a TODO list of mine. It'll be tackled in the 0.7.x release line, which is designed to simplify configuration.

Right now the only thing that comes close to this is default_achievements.ini in My Mods/SpeicalK, the values there will replace their corresponding values in per-game profiles if blank in the per-game settings. I'll have a better solution worked out in a few releases.

**Comment by SwooshyCueb** [comment]
I see we've been taking steps towards this with the new OSD. Is it still possible, with the new OSD, to specify per-profile settings? I like to have my OSD in different locations for different games, and while some games seem to need the AutoPump setting for an accurate FPS counter, others will break if this is turned on.
